code/modules/food/food/snacks.dm

```diff
@@ -27,7 +27,7 @@
 /obj/item/weapon/reagent_containers/food/snacks/Initialize()
 	. = ..()
 	if(nutriment_amt)
-		reagents.add_reagent("nutriment",nutriment_amt,nutriment_desc)
+		reagents.add_reagent("nutriment",(nutriment_amt*2),nutriment_desc)
```

Comment:
Should this not be marked with vorestation edit?

Reply:
Oh whoopsie daisy mamma mia